### DISCUSSION TITLE
Deleted WordPress admin in laragom

### DISCUSSION THREAD

**Discussion by Kaycee16A** [community/community#120066] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I mistaken deletèd the admin login account from my wordpress on laragon. How can i create a new one from the backend? Or how can i restore the deleted admin account? 

I installed a school administration plugin and tried creating a new admin specifically for that plugin. Then I realised that it was not necessary since its the same email address.  So I deleted it and since it had the same email address as the admin account,  the admin account also go deleted and I was logged out.

**Comment by ebndev** [comment]
Hi @Kaycee16A,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this, such as the Wordpress support team.

GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this.

Best of luck!
